plugins/catalog-import/src/components/types.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ConfigApi } from '@backstage/core';
+
+export type ImportOptions = {
+  pullRequest?: {
+    disable?: boolean;
+    preparePullRequest?: (
+      apis: StepperApis,
+    ) => { title?: string; body?: string };
+  };
+};
+
+export type StepperApis = {
+  configApi: ConfigApi;
```

Comment:
(I see this is not your code, so just putting this here as a side thought, no action needed)

Initial reaction when seeing this: do they really need a config API, or do they need a `Config`?

I know that one type is aliased to the other, but in this case as a reader I kinda expected this to be more on the form `type StepperOptions = { config: Config }` or so, unless the stepper _itself_ also was an API that depended on other APIs.